kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/utils/KieJarBuildHelper.java

```diff
@@ -55,7 +62,7 @@ public static void createKieJar(String resource) {
         String pomFileName = MavenRepository.toFileName(kjar.getReleaseId(), null) + ".pom";
         File pomFile = new File(System.getProperty("java.io.tmpdir"), pomFileName);
         try (FileOutputStream fos = new FileOutputStream(pomFile)) {
-            fos.write(Files.toByteArray(new File(resource + "/pom.xml")));
+            fos.write(com.google.common.io.Files.toByteArray(new File(resource + "/pom.xml")));
```

Comment:
Rather than the google utility, use JDK Files.readAllBytes (Path path)

Reply:
done